### ISSUE TITLE
[Accessibility : Keyboard & Navigation]  moving to next item in "Actions" menu using "up and down key" features are missing

### ISSUE THREAD

**Issue by Nagamani71** [kimchi-project/kimchi#914]
![accessibility23](https://cloud.githubusercontent.com/assets/17590096/14145162/713d8ee8-f6b0-11e5-9236-1b5dcddb3caa.PNG)
